doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>You can use full variable names to alleviate any uncertainty. Things aren't clear to everyone. For me, kb would be clearer than boltmk but as you never introduce the fact that you will talk about Boltzmann, I would recommend just writing out boltzmann_constant or boltzmann_k. Maybe also add comments after each line highlight what unit conversions you are doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>